src/actions/arcActions.ts

```diff
@@ -10,7 +11,12 @@ import * as Redux from "redux";
 import { ThunkAction } from "redux-thunk";
 import { Web3 } from "web3";
 
+const contributionRewardArtifacts = require('@daostack/arc.js/migrated_contracts/ContributionReward.json');
```

Comment:
This looks a bit fragile. if you just need the abi can't you get it from the arc.js instance (the "contributionrewardinstancewrapper" or whatever it is called)?

Reply:
I'll look into using the truffle contract instance, if that works well then I can remove this line entirely.